src/Teams/Repository/TeamRepository.cs

```diff
@@ -30,12 +30,12 @@ public IQueryable<Team> GetAll()
             return _dbContext.Team;
         }
 
-        public async Task<Team> GetByIdAsync(int Id) => await _dbContext.Team.FindAsync(Id);       
+        public async Task<Team> GetByIdAsync(int Id) => await _dbContext.Team.FindAsync(Id);
 
         public async Task<bool> InsertAsync(Team entity)
         {
             var obj = await _dbContext.Team.AddAsync(entity);
-            var result= obj.State == EntityState.Added ? true : false;
+            var result = obj.State == EntityState.Added ? true : false;
             await _dbContext.SaveChangesAsync();
             return result;
```

Comment:
_dbContext.SaveChangesAsync(); will return count of affected rows, so you can use check if the result is greater than 0 then insert was successful